### ISSUE TITLE
Mac 2.9.4 version crashes when opening Wallet created in 2.9.3

### ISSUE THREAD

**Issue by CirrusThink** [fyookball/electrum#243]
Should I move everything out for the fork until this is solved ?

Nov 12 13:16:10  Electron-Cash[29367]: objc[29367]: Class FIFinderSyncExtensionHost is implemented in both /System/Library/PrivateFrameworks/FinderKit.framework/Versions/A/FinderKit (0x7fffacd5da70) and /System/Library/PrivateFrameworks/FileProvider.framework/OverrideBundles/FinderSyncCollaborationFileProviderOverride.bundle/Contents/MacOS/FinderSyncCollaborationFileProviderOverride (0x114e08cd8). One of the two will be used. Which one is undefined.

Nov 12 13:16:10 Electron-Cash[29367]: assertion failed: 17B48: libxpc.dylib + 73297 [58676E88-35FD-3CF3-99BB-2A82E09F8058]: 0x89

BUG in libdispatch client: kevent[mach_recv] monitored resource vanished before the source cancel handler was invoked

**Comment by fyookball** [comment]
you might want to just create new wallet from the seed again in 2.9.4

was it a 2fa wallet?

**Comment by CirrusThink** [comment]
no I dont think it was

**Comment by CirrusThink** [comment]
i am just not sure which seed options to use

**Comment by CirrusThink** [comment]
BIP 39 only option that works but I then need to tell it what address to use, this may be same error in another ticket you have

**Comment by CirrusThink** [comment]
OK so yes to the 2fawallet , and also I tried to send some coins out and I cant even seem to be able to send coins out, 16: mandatory-script-verify-flag-failed (Signature must use SIGHASH_FORKID)

**Comment by CirrusThink** [comment]
And seeding into the 2.9.4 shows no coins or history, ..... I am getting a little worried,

**Comment by CirrusThink** [comment]
I created new wallet from seed so that I can turn off the 2auth but not it just stating Please Wait for about 30m ... its this normal process should I keep waiting or do I need to cancel this process ?

**Comment by fyookball** [comment]
well we don't support 2fa... we never did ..because trusted key doesnt allow BCH tx sig... can you try to follow instructions on #41 to put your funds into a non 2fa wallet?  You will need to use version 2.9.3 for that. once that is done you can use 2.9.4

**Comment by CirrusThink** [comment]
Just to clarify, follow the instructions to use the electron app in order to fix this ?

**Comment by CirrusThink** [comment]
Ok so I followed the directions but as a result, the new wallet does open with 2.9.3 and not 2.9.4 
Also it till easy in the title [2fa] does this mean it still carries the 2fa ??? where else can I check that 2fa is not enabled on this wallet ?

**Comment by CirrusThink** [comment]
Ok so it worked as i can send money out, but cant use the 2.9.4 versions ... sorry

**Comment by fyookball** [comment]
read the last step:

You can now use your coins as you wish. I recommend going to the Send tab and send ALLL your Bitcoin Cash to a new wallet you control.

You should send this to a newly created standard wallet made in 2.9.4.

**Comment by citrusBerry** [comment]
I have the same issue. I can access my bch in 2.9.2. But When I opened it in 2.9.4, after I typed my password, it crashed. So I need to send my bch using 2.9.2 to new wallet made in 2.9.4? Is it correct?
Since the fork is happening, I do not know if this is safe.
